R/SingleR.visualizations.R

```diff
@@ -0,0 +1,121 @@
+#' Plot a scatter plot of a single cell vs. a single reference cell/sample
+#'
+#' @param sc_data Similar to SinlgeR test input, a numeric matrix of single-cell expression values (usually
```

Comment:
I don't think there's any need to say "Similar to SingleR" here, as this suggests that users need to read the `SingleR()` documentation in order to use this function. If you want, you can add a `@details` section to describe the relationship between this function and `SingleR()`. This should be made relatively easy by having the same argument names.

Reply:
Removed.